docs/contributors/getting-started.md

```diff
@@ -132,13 +137,10 @@ To use Prettier, you should install the [Prettier - Code formatter extension](ht
     "editor.defaultFormatter": "esbenp.prettier-vscode",
     "editor.formatOnSave": true
 },
+"prettier.configPath": "packages/scripts/config/.prettierrc.js"
 ```
 
-This will use the `.prettierrc.js` file in the root folder of the Gutenberg repository and the version of Prettier that is installed in the root `node_modules` folder. To test it out prior to PR #18048 being merged, you should:
-
-1. git switch add/prettier
-2. npm ci
-3. Edit a JavaScript file and on save it will format it as defined
+This will use the `.prettierrc.js` file included in the scripts package of the Gutenberg repository.
```

Comment:
We can add this link at the beginning on the section to cover other editors:
https://prettier.io/docs/en/editors.html

Reply:
:+1: Added